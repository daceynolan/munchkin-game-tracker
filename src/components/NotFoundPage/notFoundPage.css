.page-not-found__section {
  text-align: center;
}

.page-not-found__heading {
  margin: 0;
}

.page-not-found__title {
  font-size: 26px;
  margin-bottom: 30px;
}

.page-not-found__error {
  font-size: 20px;
  text-align: center;
}

.page-not-found__content {
  font-size: 20px;
  margin-bottom: 30px;
  margin-top: 10px;
  text-align: center;
}

.page-not-found__img {
  height: 200px;
  width: 200px;
  display: block;
  margin: 0 auto;
}
