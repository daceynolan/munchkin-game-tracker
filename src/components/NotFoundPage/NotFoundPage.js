import React from "react";
import { Link } from "react-router-dom";

import "./notFoundPage.css";
import Button from "../Button";
import Monster from "../../images/monster-404.png";

const NotFoundPage = () => {
  return (
    <div className="page-not-found">
      <div className="page-not-found__section">
        <h1 className="page-not-found__heading">Level 404</h1>
        <div className="page-not-found__title">Error Monster</div>
        <div className="page-not-found__error">
          When the error monster appears, it eats the page you are looking for.
        </div>
      </div>
      <img className="page-not-found__img" src={Monster} alt="door"></img>
      <div className="page-not-found__content">
        Bad Stuff: You must run away.
      </div>
      <Button as={Link} to="/">
        Run Away
      </Button>
    </div>
  );
};

export default NotFoundPage;
