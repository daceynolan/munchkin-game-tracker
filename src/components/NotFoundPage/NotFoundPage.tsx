import React from "react";
import { Link } from "react-router-dom";

import "./notFoundPage.css";
import Button from "components/Button";
import Monster from "images/monster-404.png";

const NotFoundPage = () => {
  return (
    <div className="page-not-found">
      <div className="page-not-found__section">
        <h3 className="page-not-found__heading">Level 404</h3>
        <div className="page-not-found__title">Monster That Eats Your Page</div>
        <div className="page-not-found__error">
          He has the munchies and eats the page you are looking for. You must
          runaway and let him keep his treasure.
        </div>
      </div>
      <img className="page-not-found__img" src={Monster} alt="door"></img>
      <div className="page-not-found__content">
        <span className="page-not-found--bold">Bad Stuff: </span> You don't know
        where you are and you suffer from confusion. You are sent back home.
      </div>
      <Button as={Link} to="/">
        Run Away
      </Button>
      <div className="page-not-found__footer">10 Treasures</div>
    </div>
  );
};

export default NotFoundPage;
