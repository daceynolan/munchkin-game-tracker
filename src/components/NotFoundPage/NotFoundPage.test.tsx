import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import NotFoundPage from "./";

describe("<NotFoundPage/>", () => {
  test("renders without crashing", () => {
    const { container } = render(
      <MemoryRouter>
        <NotFoundPage />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
