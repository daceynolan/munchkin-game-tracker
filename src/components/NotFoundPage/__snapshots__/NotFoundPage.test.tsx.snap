// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage/> renders without crashing 1`] = `
<div
  class="page-not-found"
>
  <div
    class="page-not-found__section"
  >
    <h3
      class="page-not-found__heading"
    >
      Level 404
    </h3>
    <div
      class="page-not-found__title"
    >
      Monster That Eats Your Page
    </div>
    <div
      class="page-not-found__error"
    >
      He has the munchies and eats the page you are looking for. You must runaway and let him keep his treasure.
    </div>
  </div>
  <img
    alt="door"
    class="page-not-found__img"
    src="monster-404.png"
  />
  <div
    class="page-not-found__content"
  >
    <span
      class="page-not-found--bold"
    >
      Bad Stuff: 
    </span>
     You don't know where you are and you suffer from confusion. You are sent back home.
  </div>
  <a
    class="button button--default"
    href="/"
  >
    Run Away
  </a>
  <div
    class="page-not-found__footer"
  >
    10 Treasures
  </div>
</div>
`;
